values.yaml

```diff
@@ -0,0 +1,26 @@
+# https://github.com/kubernetes/ingress-nginx/blob/helm-chart-4.0.13/charts/ingress-nginx/values.yaml
+controller:
+  resources:
+    requests:
+      cpu: 250m
+      memory: 512Mi
```

Comment:
I am uncertain whether we should make these values configurable or allow module users to add other values files.

cc @Monska85